Assets/AdditionalScripts/GlobalScripts/Global.cs

```diff
@@ -77,38 +77,57 @@ void checkKilledChickens()
 
-            StartCoroutine(waitResetAndPlayAgain());
+            StartCoroutine(WaitResetAndPlayAgain());
         }
     }
 
-    void pickRandomQuestion()
+    void PickRandomQuestion()
     {
-        int randomNumber  = Random.Range(1,10);
-
-        if(randomNumber == 1) {
-            updateSignAndChickens("Q1", "R1", "W1.1", "W1.2", "W1.3", "W1.4");
-        } else if (randomNumber == 2){
-            updateSignAndChickens("Q2", "R2", "W2.1", "W2.2", "W2.3", "W2.4");
-        } else if (randomNumber == 3){
-            updateSignAndChickens("Q3", "R3", "W3.1", "W3.2", "W3.3", "W3.4");
-        } else if (randomNumber == 4){
-            updateSignAndChickens("Q4", "R4", "W4.1", "W4.2", "W4.3", "W4.4");
-        } else if (randomNumber == 5){
-            updateSignAndChickens("Q5", "R5", "W5.1", "W5.2", "W5.3", "W5.4");
-        } else if (randomNumber == 6){
-            updateSignAndChickens("Q6", "R6", "W6.1", "W6.2", "W6.3", "W6.4");
-        } else if (randomNumber == 7){
-            updateSignAndChickens("Q7", "R7", "W7.1", "W7.2", "W7.3", "W7.4");
-        } else if (randomNumber == 8){
-            updateSignAndChickens("Q8", "R8", "W8.1", "W8.2", "W8.3", "W8.4");
-        } else if (randomNumber == 9){
-            updateSignAndChickens("Q9", "R9", "W9.1", "W9.2", "W9.3", "W9.4");
-        } else if (randomNumber == 10){
-            updateSignAndChickens("Q10", "R10", "W10.1", "W10.2", "W10.3", "W10.4");
+        int randomNumber = Random.Range(1, 10);
+
+        if (randomNumber == 1)
+        {
+            UpdateSignAndChickens("Q1", "R1", "W1.1", "W1.2", "W1.3", "W1.4");
+        }
+        else if (randomNumber == 2)
+        {
+            UpdateSignAndChickens("Q2", "R2", "W2.1", "W2.2", "W2.3", "W2.4");
+        }
+        else if (randomNumber == 3)
+        {
+            UpdateSignAndChickens("Q3", "R3", "W3.1", "W3.2", "W3.3", "W3.4");
+        }
+        else if (randomNumber == 4)
+        {
+            UpdateSignAndChickens("Q4", "R4", "W4.1", "W4.2", "W4.3", "W4.4");
+        }
+        else if (randomNumber == 5)
+        {
+            UpdateSignAndChickens("Q5", "R5", "W5.1", "W5.2", "W5.3", "W5.4");
+        }
+        else if (randomNumber == 6)
+        {
+            UpdateSignAndChickens("Q6", "R6", "W6.1", "W6.2", "W6.3", "W6.4");
+        }
+        else if (randomNumber == 7)
+        {
+            UpdateSignAndChickens("Q7", "R7", "W7.1", "W7.2", "W7.3", "W7.4");
+        }
+        else if (randomNumber == 8)
+        {
+            UpdateSignAndChickens("Q8", "R8", "W8.1", "W8.2", "W8.3", "W8.4");
+        }
+        else if (randomNumber == 9)
+        {
+            UpdateSignAndChickens("Q9", "R9", "W9.1", "W9.2", "W9.3", "W9.4");
+        }
+        else if (randomNumber == 10)
+        {
+            UpdateSignAndChickens("Q10", "R10", "W10.1", "W10.2", "W10.3", "W10.4");
         }
```

Comment:
Qualitywise, I strongly disagree with this paragraph, even though it is already merged.
Please refactor it.
```suggestion
        UpdateSignAndChickens(string.Format("Q{0}", randomNumber), string.Format("R{0}", randomNumber), string.Format("W{0}.1", randomNumber), string.Format("W{0}.2", randomNumber), string.Format("W{0}.3", randomNumber), string.Format("W{0}.4", randomNumber));
```